README.md

```diff
@@ -18,4 +18,7 @@ This is especially important to check the feasibility of the project. The major
 - File conversion
 
 ### Future scope
-This can be discussed at a later point.
\ No newline at end of file
+This can be discussed at a later point.
+
+## How to add in changes?
+To add in any changes, please create a new branch and raise an MR targeted to master. This will help in version control and reduce confusion.
```

Comment:
An example of how this can be used is this commenting feature. This will be done after discussion on whether the change is eligible to be added to main or not.